src/MissionCreator.sol

```diff
@@ -78,7 +78,8 @@ contract MissionCreator is Ownable, IERC721Receiver {
         address payable moonDAOTreasuryPayable = payable(moonDAOTreasury);
         IJBTerminal terminal = IJBTerminal(jbMultiTerminalAddress);
 
-        //TODO: Configure ruleset
+        uint256 deadline = block.timestamp + 30 days;
```

Comment:
Just a thought, at MoonDAO we did 1 lunar cycle and that was fun because if you start on a full moon you'll end the raise on the next full moon, so that would be 28 days. Maybe let's stick with the moon cycles for moondao?

Reply:
of course